backend/globaleaks/jobs/backup.py

```diff
@@ -0,0 +1,102 @@
+from datetime import datetime, timedelta
+import os
+import sqlite3
+import subprocess
+
+
+from globaleaks.utils.backup import get_backups_parameter, reset_audit_log_file
+from globaleaks import models
+from globaleaks.orm import db_log, transact
+from globaleaks.utils.utility import datetime_now
+from globaleaks.jobs.job import LoopingJob
+from globaleaks.settings import Settings
+from globaleaks.utils.log import log
+
+from twisted.internet.defer import inlineCallbacks
+
+
+__all__ = ['Backup']
+
+
+def run_rsync(session, source_path, destination_path):
+    try:
+        rsync_command = ["rsync", "-av", source_path, destination_path]
+        subprocess.run(rsync_command, check=True)
+    except Exception as e:
+        db_backup_log(session, e)
+
+
+def backup_sqlite_database_and_attachments(session, backup_db_path, backup_attachments_path):
+    run_rsync(session, Settings.attachments_path, backup_attachments_path)
```

Comment:
@joeman65 @alessiofranceschini: This call to rsync seems to be run with a database transaction and will likely block any access during the backup. Supposing the platform in a day have received significan information (e.g. 10GB of data)  this might block for hours.
We consider the stategy should consider making a first rsync out of the transaction and then again a second within the transaction.